### DISCUSSION TITLE
Welcome to docs Discussions!

### DISCUSSION THREAD

**Discussion by WojciechSkirlo** [WojciechSkirlo/docs#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WojciechSkirlo** [comment]
Hi, We have to standardize the sentence `Try it in the Playground` because I saw a lot of versions for example 
`Spróbój tego w piaskownicy`, `Spróbuj w piaskownicy`, `Spróbój w praktyce`, `Spróbuj tego w Vue SFC Playground`.

Have a nice day

**Reply by andrzejrumak** [reply]
What version do you suggest? I'll use one version. I would choose "Spróbuj tego w Vue Playground".

**Reply by WojciechSkirlo** [reply]
Yess It can be. I also think about this version.

**Reply by andrzejrumak** [reply]
OK I have changed this in to. one version and push in feature/lifecyclehooks  brunch
Have a good day

**Reply by andrzejrumak** [reply]
Hello Wojciech. How are you doing? Sorry I was busy but we can continue the translation.  Could you merge the braches [feature/componentsbasic], [feature/watchers] to main, please? Andrzej
